### PR TITLE
Footer and continue button

### PR DESCRIPTION
**Pull request by accenturepavelsgarklavs**

### CODE REVIEW

File: UnlimitedAccess/Controllers/MainViewController/MainViewController.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            make.leading.equalTo(view.safeAreaLayoutGuide).offset(40)
+            make.trailing.equalTo(view.safeAreaLayoutGuide).inset(40)
+        }
+    }
+
+    func setupFooterStack() {
+        view.addSubview(footerStack)
+
+        // Usage only to fill in stack view and not writing three separate labels.
+        for data in TaskManager.shared.footerData {
```
Comment (eugenemokhan) [inline UnlimitedAccess/Controllers/MainViewController/MainViewController.swift:59]: what is TaskManager?

File: UnlimitedAccess/TaskManager.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
+//
+// Created by pavels.garklavs on 03/03/2022.
+//
+
+import Foundation
+
+final class TaskManager {
```
Comment (eugenemokhan) [inline UnlimitedAccess/TaskManager.swift:7]: let's rename it

**Review by eugenemokhan — APPROVED** [review]

File: UnlimitedAccess/Controllers/MainViewController/MainViewController.swift (hunk 1 above)
Comment (accenturepavelsgarklavs) [reply UnlimitedAccess/Controllers/MainViewController/MainViewController.swift:59]: Where I store some "dummy" data to use.

File: UnlimitedAccess/TaskManager.swift (hunk 2 above)
Comment (accenturepavelsgarklavs) [reply UnlimitedAccess/TaskManager.swift:7]: To what should I rename it?

File: UnlimitedAccess/Controllers/MainViewController/MainViewController.swift (hunk 1 above)
Comment (eugenemokhan) [reply UnlimitedAccess/Controllers/MainViewController/MainViewController.swift:59]: idk, but it isn't taskManager

File: UnlimitedAccess/TaskManager.swift (hunk 2 above)
Comment (eugenemokhan) [reply UnlimitedAccess/TaskManager.swift:7]: idk, but it isn't taskManager

Comment (accenturepavelsgarklavs) [reply UnlimitedAccess/TaskManager.swift:7]: Okay, will think of something and rename it. Thanks!

**Pull request closed without merging**